src/js/no-react-entry.js

```diff
@@ -5,6 +5,7 @@ import createApplicationStatus from './common/components/createApplicationStatus
 const wizardPages = new Set(['/education/apply/', '/education/eligibility/']);
 const pensionPages = new Set(['/pension/', '/pension/apply/', '/pension/eligibility/']);
 const healthcarePages = new Set(['/health-care/', '/health-care/apply/', '/health-care/eligibility/']);
+const burialPages = new Set(['/burials-and-memorials/', 'burials-and-memorials/eligibility/']);
```

Comment:
There is not currently an apply page, should one be created? Should that be part of this set of changes? Or should a ticket be written for this?